tools/distpartitioning/data_proc_pipeline.py

```diff
@@ -44,38 +34,15 @@ def log_params(params):
                      help='The schema of the graph')
     parser.add_argument('--num-parts', required=True, type=int,
                      help='The number of partitions')
-    parser.add_argument('--num-node-weights', required=True, type=int,
-                     help='The number of node weights used by METIS.')
-    parser.add_argument('--workspace', type=str, default='/tmp',
-                    help='The directory to store the intermediate results')
```

Comment:
you don't need this any more?

Reply:
The unused parameters are eliminated. Some of these parameters where used when single-file-format was supported. But today we plan to only support multiple machines using multiple-file-format dataset. This was also discussed in our chime call as well.